File: Web/EYE/EYE/Model/DAO/StatementSlack.cs
using System.Data.SqlClient;

namespace EYE.Model.DAO
{
    public class StatementSlack
    {
        private const int MINIMO_DE_ALTERACAO = 1;
        public static bool InserirSlack(Slack slack)
        {
            using (var conexao = Conexao.GetConexao())
            {
                using (SqlCommand cmd = new SqlCommand("UPDATE workspace SET url_slack = @url_slack WHERE cod_workspace = @cod_workspace", conexao))
                {
                    cmd.Parameters.AddWithValue("@url_slack", slack.Url);
                    cmd.Parameters.AddWithValue("@cod_workspace", slack.CodWorkspace);
					return cmd.ExecuteNonQuery() >= MINIMO_DE_ALTERACAO;
                }
            }
        }
		public static string VerificaUrlCadastrada(int codWorkspace)
		{
			using (var conexao = Conexao.GetConexao())
			{
				using (SqlCommand cmd = new SqlCommand("SELECT url_slack FROM workspace WHERE cod_workspace = @cod_workspace", conexao))
				{
					cmd.Parameters.AddWithValue("@cod_workspace", codWorkspace);
					using (SqlDataReader leitor = cmd.ExecuteReader())
					{
						return (leitor.Read()) ? leitor.GetString(0) : null;
					}
				}
			}
		}
	}
}

File: Local/Eye/src/main/java/br/com/eye/model/dao/StatementSlack.java
package br.com.eye.model.dao;

import br.com.eye.model.SlackMensagem;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;

public class StatementSlack {

    public static SlackMensagem getSlack(int codWorkspace) throws SQLException {
        SlackMensagem slack = new SlackMensagem();
        String sql = "SELECT url_slack FROM workspace WHERE cod_workspace = ?";
        PreparedStatement query = new Conexao().getConexao().prepareStatement(sql);
        query.setInt(1, codWorkspace);
        ResultSet resultado = query.executeQuery();
        if (resultado.next()) {
        slack.setUrl(resultado.getString("url_slack"));          
        }
        return slack;
    }
}
